demisto_sdk/commands/common/errors.py

```diff
@@ -443,6 +444,18 @@ def missing_get_mapping_fields_command():
         return 'The command "get-mapping-fields" is missing from the YML file and is required as the ismappable ' \
                'field is set to true.'
 
+    @staticmethod
+    @error_code_decorator
+    def readme_missing_output_context(command, context_paths):
+        return f'The Following context paths for command {command} are found in YML file ' \
+               f'but are missing from the README file: {context_paths}'
+
+    @staticmethod
+    @error_code_decorator
+    def missing_output_context(command, context_paths):
+        return f'The Following context paths for command {command} are found in the README file ' \
+               f'but are missing from the YML file: {context_paths}'
+
```

Comment:
dup
```suggestion
```

Reply:
I don't see dup in my version- one is for readme and not YML and one is for YML and not readme.